mapper/pojo-standalone/src/main/java/org/hibernate/search/mapper/pojo/standalone/work/SearchWorkspace.java

```diff
@@ -77,6 +77,7 @@ public interface SearchWorkspace {
 	 * These operations will not be applied immediately just because  a call to {@link #flush()} is issued:
 	 * the "flush" here is a very low-level operation handled by the backend.
 	 */
+	// TODO: Yoann, does it make sense to talk about "automatic indexing and transaction being committed" in a context of a pojo mapper?
```

Comment:
It does not, feel free to remove the paragraph starting with "Note that some operations [...]".